base/client.jl

```diff
@@ -442,11 +442,11 @@ function run_fallback_repl(interactive::Bool)
                 eval_user_input(stderr, ex, true)
             end
         else
+            if interactive
+                print("julia> ")
+                flush(stdout)
+            end
             while !eof(input)
```

Comment:
Thanks. I would suggest DRY the code slightly with break, but otherwise lgtm:
```suggestion
          while true
              if interactive
                  print("julia> ")
                  flush(stdout)
              end
              eof(input) && break
```